### PR TITLE
Expose sendRawTransaction

### PR DESCRIPTION
**Pull request by MirekR**
## Description
- Adding option to send raw transaction for cases where you already have fully defined transactions.
Example workflow where sendRawTransaction is: 1. Request partial signed transaction from a server, 2. Sign transaction as user 3. send transaction. 

## Work Completed
- Added sendRawTransaction function
- Refactor sendTransaction function to use sendRawTransaction to avoid code duplication

### CODE REVIEW

**Review by ajamaica — COMMENTED** [review]
@MirekR thanks a lot for your contribution. We are about to deprecate all the decodeTo methods in favor for a much better serialization away from moshi. moshi is not great since it requires pre configuration as you might know. Most of the rx functions are not affected. The change is completed waiting for review. This will affect this functions. 

I recommend wait for those changes since they will affect the code that you nicely provide to the project. 

@Funkatronics we need to get that merge.

**Comment by MirekR** [comment]
Cool, I'll wait until you merge your change and will get back to this later. Thanks :)

**Comment by ajamaica** [comment]
It's now merged. Thanks again for your contribution

**Comment by MirekR** [comment]
Alreet, I merged your code and updated my changes but I'm running in issues in tests ... 

TestGetConfirmedBlock with exception 

`Transaction version (0) is not supported by the requesting client. Please try the request again with the following configuration parameter: "maxSupportedTransactionVersion": 0
`
Have you seen that?

**Comment by MirekR** [comment]
It looks like the latest block is already containing version transaction what is not supported yet (unless I'm missing something). 

I've updated tests to use fixed slot for test for now. Does that sounds ok to you?

**Comment by ajamaica** [comment]
That's ok but then we need another issue for reading this

**Comment by ajamaica** [comment]
@MirekR lets file a new issue and solve that on a new PR

**Comment by MirekR** [comment]
Yeap, I've created new issue for it - https://github.com/metaplex-foundation/SolanaKT/issues/112 

Thanks, @ajamaica and btw, the new serialisation looks good, great work.

**Comment by ajamaica** [comment]
It was a big effort

**Comment by MirekR** [comment]
It was :) 

Is there anything else in this PR you'd like me to change or you happy to merge this?

File: solana/src/main/java/com/solana/api/sendRawTransaction.kt
Comment (ajamaica) [inline solana/src/main/java/com/solana/api/sendRawTransaction.kt:61]: Thanks!

**Review by ajamaica — APPROVED** [review]

**Comment by ajamaica** [comment]
@MirekR I want the other PR https://github.com/metaplex-foundation/SolanaKT/pull/110 and then this one

**Comment by ajamaica** [comment]
We merge the changes. There is a conflict for sendTransaction. I think is only the imports. Is it too much to ask if you rebase?

**Comment by MirekR** [comment]
That's now conflicts resolved :)

**Comment by ajamaica** [comment]
This can go in thanks a lot

**Review by ajamaica — APPROVED** [review]

**Pull request merged**
